web/app.js

```diff
@@ -2434,6 +2439,29 @@ function webViewerKeyDown(evt) {
   }
 }
 
+/**
+ * Converts API PageLayout values to the format used by `PDFViewer`.
+ * NOTE: This is supported to the extent that the viewer implements the
+ *       necessary Scroll/Spread modes (since SinglePage, TwoPageLeft,
+         and TwoPageRight all suggests using non-continuous scrolling).
```

Comment:
It looks like an asterisk is missing here at the beginning of the line.

Reply:
D'oh, fixed now :-)

Thinking about this patch I suppose that it might surprise users when some documents now opens with Spread mode enabled, but given that this is already controllable via existing prefs[1] I really didn't want to add any new ones here.

---
In Firefox, setting `pdfjs.spreadModeOnLoad = 0` in `about:config` is now guaranteed to never enable any Spread mode by default.